doc/admin-guide/files/records.yaml.en.rst

```diff
@@ -783,6 +786,9 @@ mptcp
 
    Requires custom Linux kernel available at https://multipath-tcp.org.
 
+allow-plain
+   For TLS ports, will fall back to non-TLS processing if the TLS handshake fails.
```

Comment:
We may want to mention that this is for TLS over TCP (or ports configured with `ssl`), although plain QUIC connection does not exist in the first place. We could also say this is incompatible with `quic` like some other descriptors say.

Reply:
Fails for any reason, including say an expired certificate? Or does this require a protocol failure?

Reply:
Allow-plain only kicks in of the first packet fails to be recognized as a valid client-hello. Once packets are exchanged in the TLS path, it is no reasonable to try an reinterpret as HTTP.

Reply:
Will add comments to the effect that allow-plain does not make sense for quic.  Could also implement an incompatible check.